backend/project/__init__.py

```diff
@@ -0,0 +1,34 @@
+#### BACKEND PROJECT PACKAGE INITIALIZATION ####
+
```

Comment:
I noticed that we've been using a lot of spacing in these files. Not sure if we want to keep that whitespace or condense it now that the code is written out.